README.rst

```diff
@@ -32,21 +32,49 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
-Tutorial
---------
+Quick Start
```

Comment:
recommendation - create sub-headings for Prerequisites (python 3.4 etc), Downloading and Installing, and Accessing/Viewing (this already has a heading "python in the browser"), for easier skimming/chunking.